ui/app/rspack.config.mjs

```diff
@@ -11,16 +11,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import { dirname, resolve } from 'node:path';
+import { fileURLToPath } from 'node:url';
 import rspack from '@rspack/core';
 import refreshPlugin from '@rspack/plugin-react-refresh';
 import TerserPlugin from 'terser-webpack-plugin';
 import { defineConfig } from '@rspack/cli';
 
+const __dirname = dirname(fileURLToPath(import.meta.url));
```

Comment:
if we are using mjs, we probably can use `import.meta.dirname` directly

Reply:
:+1:

using TS didn't work because of
```
rspack.config.ts:24:19 - error TS1343: The 'import.meta' meta-property is only allowed when the '--module' option is 'es2020', 'es2022', 'esnext', 'system', 'node16', or 'nodenext'.
```